src/scripts/extensions/clipperInject.ts

```diff
@@ -192,7 +192,7 @@ export class ClipperInject extends FrameInjectBase<ClipperInjectOptions> {
 			label: Log.Failure.Label.UnhandledExceptionThrown,
 			properties: {
 				failureType: Log.Failure.Type.Unexpected,
-				failureInfo: { error: JSON.stringify({ error: e.toString(), url: window.location.href }) },
```

Comment:
Should we remove his JSON structure from the failureInfo? We might have dashboards/scripts that rely on the Failure info having the specified structure